pulsar-client-tools/src/main/java/com/yahoo/pulsar/admin/cli/CmdPersistentTopics.java

```diff
@@ -248,6 +252,20 @@ void run() throws PulsarAdminException {
         }
     }
 
+    @Parameters(commandDescription = "Get the internal metadata info for the topic")
+    private class GetInternalInfo extends CliCommand {
+        @Parameter(description = "persistent://property/cluster/namespace/destination\n", required = true)
+        private java.util.List<String> params;
+
+        @Override
+        void run() throws PulsarAdminException {
+            String persistentTopic = validatePersistentTopic(params);
```

Comment:
`params` is a list of string arguments. `validatePersistentTopic()` is being used for all the commands to extract the topic name and ensure there are no other trailing arguments.